src/main/java/gov/wa/wsdot/mobile/shared/YouTubeFeed.java

```diff
@@ -23,38 +23,44 @@
 
 public class YouTubeFeed extends JavaScriptObject {
```

Comment:
I need to look more into that and figure it out. I've found it tough to wrap my mind around what is going on with JsInterop and the different annotations.